docs/compatible_configs/pycodestyle/.flake8

```diff
@@ -0,0 +1,3 @@
+[pycodestyle]
+max-line-length = 88
+ignore = E203,E701
```

Comment:
pycodestyle does not read the `.flake8` config file

this configuration also will result in quite a lot of frustration as it enables several pairs of incompatible codes

Reply:
ah, i just added the files based off what was previously documented in black

what further incompatible codes need to be disabled?

Reply:
personally I would include (at least) all of the disabled-by-default codes